en/micro-integrator/docs/references/connectors/google-spreadsheet-connector/google-spreadsheet-connector-example.md

```diff
@@ -0,0 +1,387 @@
+                <respond/>
+            </inSequence>
+            <outSequence/>
+            <faultSequence/>
+        </resource>
+        <resource methods="POST" uri-template="/read">
+            <inSequence>
+                <sequence description="This sequence will read data of the spreadsheet. " key="readData"/>
+                <respond/>
+            </inSequence>
+            <outSequence/>
+            <faultSequence/>
+        </resource>
+        <resource methods="POST" uri-template="/edit">
+            <inSequence>
+                <sequence key="editSpreadsheet"/>
+                <respond/>
+            </inSequence>
+            <outSequence/>
+            <faultSequence/>
+        </resource>
+    </api>
+
+```
+
+{!references/connectors/exporting-artifacts.md!}
+
+
+## Get the project
+
+You can download the ZIP file and extract the contents to get the project code.
+
+<a href="../../../../assets/attach/connectors/FileConnector.zip">
+    <img src="../../../../assets/img/connectors/download-zip.png" width="200" alt="Download ZIP">
+</a>
+
+## Deployment
+
+Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 
+
+{!references/connectors/deploy-capp.md!}
+
+## Testing
+
+### Spreadsheet insert Operation
+
+1. Invoke the SpreadsheetAPI with the following URL. Application such as [Postman](https://www.postman.com/) can be used to invoke the API. 
+
+```
+  Resource method: POST
+  URL: http://localhost:8290/spreadsheet/insert
+```
+
+  ```
+    {
+      "properties":{
+          "title": "Company"
+        },
+      "sheets":[
+          {
+            "properties":
+            {
+              "title": "Employees"
+            }
+          },
+          {
+            "properties":
+            {
+              "title": "Hector"
+            }
+          }
+        ], 
+        "range":"Employees!A1:C3",
+        "values":[
+            [
+                "First Name",
+                "Last Name",
+                "Gender"
+            ],
+            [
+                "John",
+                "Doe",
+                "Male"
+            ],
+            [
+              "Leon",
+                "Wins",
+                "Female"
+              ]
+        ]
+    }
+    
+  ```
+**Expected Response**: 
+You should get a success response as below, and the spreadsheet should be created in the given ID in the response with data inserted.  
+
+```
+  {
+    "spreadsheetId": "1ddnO00fcjuLvEMCUORVjYQ4C0VLeAPNGmcvSvELHbPU",
+    "updates": {
+      "spreadsheetId": "1ddnO00fcjuLvEMCUORVjYQ4C0VLeAPNGmcvSvELHbPU",
+      "updatedRange": "Employees!A1:C3",
+      "updatedRows": 3,
+      "updatedColumns": 3,
+      "updatedCells": 9
+    }
+  }
+```
+
+### Spreadsheet Read Operation
+
+1. Invoke the SpreadsheetAPI with the following URL. Application such as [Postman](https://www.postman.com/) can be used to invoke the API. Obtain the Spreadsheet ID from the step 1. 
```

Comment:
```suggestion
Invoke the SpreadsheetAPI with the following URL. Application such as [Postman](https://www.postman.com/) can be used to invoke the API. Obtain the Spreadsheet ID from the step 1. 
```
Remove the list numbering if there is only one item. Please apply below as well.

Reply:
Done